handler_websocket.go

```diff
@@ -306,9 +306,9 @@ func (t *websocketTransport) DisabledPushFlags() uint64 {
 	return PushFlagDisconnect
 }
 
-// AppLevelPing ...
-func (t *websocketTransport) AppLevelPing() AppLevelPing {
-	return AppLevelPing{
+// PingPongConfig ...
+func (t *websocketTransport) PingPongConfig() PingPongConfig {
+	return PingPongConfig{
```

Comment:
It seems that we are loosing the fact ping/pong was disabled here. Because in the code above Websocket handler uses `pingInterval, pongTimeout := getPingPongPeriodValues(s.config.PingPongConfig)` - we are loosing `-1`. Since we are migrating to `PingPongConfig` everywhere - I suppose it should be proxied to the Client unchanged.

Reply:
@FZambia I'm a bit confused here. I suppose that both `-1` and `0` would work just as fine in `Client` because we are essentially checking `c.pingInterval > 0`, when we are deciding whether to send ping or not?

Reply:
Yeah, you are right - it will work correctly.

Though still the idea here that `PingPongConfig` may be simply proxied to the `Client` without the need to make any changes in WebSocket handler. I.e. the call to `getPingPongPeriodValues` may happen only in `Client`. Transport just returns `PingPongConfig` that was configured unchanged. This will simplify code a bit.

Reply:
@FZambia  Oh I see. But I'm wondering if we still need to keep it in `Transport` if we are not using transport level ping/pong anymore? Is it possible we just remove the `PingPongConfig` from `Transport` and move it to `centrifuge.Config`?

Reply:
I asked myself the same question - but I think better to postpone this decision. The original idea was to let users easily use different ping/pong configurations for different transports. Just moving config to Node - we will loose this property. Let's keep things for now - but keep this in mind.

Reply:
It will - right, just more `if` branches in code. Though the need for these branches may be very rare - TBH personally I can't imagine a real scenario for having different ping/pong configuration for different transports, possibly GRPC transport  requirements may be slightly different in practice 🤔 But it's definitely not common. Let me think a bit more - I'll make the change if I come to the decision.